release-calendar/test/test-data.ts

```diff
@@ -30,59 +30,59 @@ function promoteRelease(
 export default async function addTestData(
   repositoryService: RepositoryService,
 ): Promise<void> {
-  await Promise.all(
-    [
-      new Release('1234567890123'), // lts
-      new Release('2234567890123'), // TODO(estherkim): promote to stable
-      new Release('3234567890123'), // TODO(estherkim): promote to percent_*
-      new Release('4234567890123'), // TODO(estherkim): promote to opt_in_*
-      new Release('5234567890123'), // nightly
-      new Release('6234567890123'),
-    ].map(async (release) => {
-      return repositoryService.createRelease(release);
-    }),
-  );
+  const releases = [
+    new Release('1234567890123'), // lts
+    new Release('2234567890123'), // stable
+    new Release('3234567890123'), // perc-beta and perc-experimental
+    new Release('4234567890123'), // opt-in-beta and opt-in-experimental
+    new Release('5234567890123'), // nightly
+  ];
 
-  const releases = await repositoryService.getReleases();
-  const promises = [];
-  const startDate = new Date(Date.now());
+  const today = new Date();
+  const startDate = new Date(today.setMonth(today.getMonth() - 1));
```

Comment:
Looks like the releases span from `Mar 9 to Mar 29`. So when you pull up the calendar now, it will show no releases? Will it bother you instead of adding releases from `x to today`?